<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainPage</class>
 <widget class="QWidget" name="MainPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>850</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>主页</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(background.png)</string>
  </property>
  <widget class="QPushButton" name="background">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>701</width>
     <height>531</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>background.png</normaloff>background.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>1000</width>
     <height>1000</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>61</width>
     <height>521</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="cnRank_button">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>finger.png</normaloff>finger.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="worldRank_button">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>finger.png</normaloff>finger.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="chemRank_button">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>finger.png</normaloff>finger.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="teacher_button">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>finger.png</normaloff>finger.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>580</y>
     <width>681</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>大学排名与导师查询系统</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
